### PR TITLE
Adding DDB SLP

### PR DESCRIPTION
**Pull request by Harrisonhcue**
- Adding support for a new currency (DDB) on behalf of Discount Distributors Bahamas. 
- Supported extension is "com.generalbytes.batm.server.extensions.extra.slp.SlpExtension"
- Please advise if any action is required to complete this merge and list the DDB ticker.

### CODE REVIEW

File: server_extensions_extra/src/main/resources/batm-extensions.xml [hunk 1]
```diff
@@ -1990,11 +1991,11 @@
             <param name="password" />
             <param name="host" />
             <param name="port" />
-            <cryptocurrency>SPICE</cryptocurrency>
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/resources/batm-extensions.xml:1993]: Do not remove SPICE token, add a new line with your token

File: server_extensions_extra/src/main/resources/batm-extensions.xml [hunk 2]
```diff
@@ -1990,11 +1991,11 @@
...
+            <cryptocurrency>DDB</cryptocurrency>
         </wallet>
         <!-- See https://github.com/kosinusbch/slp-token-icons for token logos -->
-        <cryptologo cryptocurrency="SPICE" file="spice.png"/>
-        <cryptologo cryptocurrency="SPICE" file="spice.svg"/>
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/resources/batm-extensions.xml:1997]: Do not remove SPICE token, add a new lines with your token

File: currencies/src/main/java/com/generalbytes/batm/common/currencies/SlpToken.java [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import java.util.Map;
 
 public enum SlpToken {
-    SPICE(8, "4de69e374a8ed21cbddd47f2338cc0f479dc58daa2bbe11cd604ca488eca0ddf"),
```
Comment (pvyhnal-generalbytes) [inline currencies/src/main/java/com/generalbytes/batm/common/currencies/SlpToken.java:8]: Do not remove SPICE token, add a new line with your token

**Review by pvyhnal-generalbytes — CHANGES_REQUESTED** [review]
- SVG logos should not contain filters or bitmap images
- All SVG and PNG logos should be 120 x 120px. See other logos for examples.
- Preserve the unified BATM design: every SVG logo must contain the background from template.svg.

Your PNG logo seems to be too small and does not have transparent background, see template.png

Your SVG logo contains embedded bitmap, please submit a vector logo

File: server_extensions_extra/src/main/resources/batm-extensions.xml (hunk 1 above)
Comment (Harrisonhcue) [reply server_extensions_extra/src/main/resources/batm-extensions.xml:1993]: The latest commit addresses this!

**Comment by Harrisonhcue** [comment]
Hello, 

Requesting a status update on this PR.

Kindly advise 😃

**Comment by pvyhnal-generalbytes** [comment]
The SVG logo is too large, the PNG logo does not use the template.
Please see the template SVG and PNG files or other crypto currency logos.

**Comment by Harrisonhcue** [comment]
@pvyhnal-generalbytes Do you mean the SVG logo file size is too large? or the dimensions? 

Kindly Advise.

**Comment by pvyhnal-generalbytes** [comment]
The ddb.svg dimensions were too large but after your last commit it seems to be allright.

**Review by pvyhnal-generalbytes — APPROVED** [review]

File: currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java [hunk 4]
```diff
@@ -90,6 +91,7 @@
     SPICE("Spice"),
     START("Startcoin"),
     SUM("Sumcoin"),
+    SUNCASH("SunCash")
```
Comment (filipocelka) [inline currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java:94]: Missing comma

**Review by filipocelka — CHANGES_REQUESTED** [review]

File: currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java (hunk 4 above)
Comment (Harrisonhcue) [reply currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java]: Apologies, I've updated this accordingly.

**Review by filipocelka — APPROVED** [review]

**Pull request merged**